### PR TITLE
Added config for Boosts/MouseHighlight

### PR DESCRIPTION
**Pull request by Sethtroll**
Changed overlays to appear for "RS" style
Added Utilities class to help shadow/outline text/rect

### CODE REVIEW

**Comment by Adam-** [comment]
Thanks for your interest in the project. You do not need to close and open a new pull request to add a new commit to the PR (or to modify existing commits on the PR). Just push a new commit.

**Comment by kfricilone** [comment]
Maybe add the 'Utility' methods to this class instead of heaving multiple rendering related utility classes?

net.runelite.client.ui.overlay.OverlayUtil

File: runelite-client/src/main/java/net/runelite/client/plugins/boosts/BoostsConfig.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package net.runelite.client.plugins.boosts;
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/boosts/BoostsConfig.java]: New files should contain a license header at the top

File: runelite-client/src/main/java/net/runelite/client/plugins/boosts/BoostsOverlay.java [hunk 2]
```diff
@@ -94,11 +96,21 @@ public Dimension render(Graphics2D graphics)
...
+			String str2 = "/" + base;
+			if (base < boosted)
+			{
+				graphics.setColor(Color.GREEN);
+			}
+			else
+			{
+				graphics.setColor(Color.RED);
+			}
 			String str = boosted + "/" + base;
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/boosts/BoostsOverlay.java]: This variable is unused now? Should be removed

File: runelite-client/src/main/java/net/runelite/client/plugins/mousehighlight/MouseHighlightConfig.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2017, Adam <Adam@sigterm.info>
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/mousehighlight/MouseHighlightConfig.java]: This should be you and not me

File: runelite-client/src/main/java/net/runelite/client/plugins/boosts/BoostsConfig.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+    keyName = "boosts",
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/boosts/BoostsConfig.java]: All of these indents should be tabs

File: runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoOverlay.java [hunk 5]
```diff
@@ -117,22 +119,22 @@ public Dimension render(Graphics2D graphics)
...
 		int x = (WIDTH - fm.stringWidth(opponentName)) / 2;
 		graphics.setColor(Color.white);
-		graphics.drawString(opponentName, x, fm.getHeight() + TOP_BORDER);
+		OverlayUtil.shadowString(graphics, opponentName, x, fm.getAscent() + TOP_BORDER);
 
 		if (lastRatio >= 0)
 		{
 			int barWidth = (int) (lastRatio * (float) BAR_WIDTH);
-			int barY = TOP_BORDER + fm.getHeight() + 1;
+			int barY = TOP_BORDER + fm.getHeight() + TOP_BORDER;
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoOverlay.java]: What is the purpose of this change?
